web/src/settings_data.ts

```diff
@@ -141,6 +142,16 @@ export function user_can_invite_others_to_realm(): boolean {
     return user_has_permission(page_params.realm_invite_to_realm_policy);
 }
 
+export function user_can_create_muliuse_invite(): boolean {
+    if (!page_params.user_id) {
+        return false;
+    }
+    return user_groups.is_user_in_group(
+        page_params.realm_create_multiuse_invite_group_id,
+        page_params.user_id,
+    );
+}
```

Comment:
I initially wrote the function like this 
```
export function user_can_create_muliuse_invite(): boolean {
    return user_groups.is_user_in_group(
        page_params.realm_create_multiuse_invite_group_id,
        people.my_current_user_id()
    );
}
```
But `People` module can not be imported because of circular dependancy.